test/unit/components/chart/daily.test.js

```diff
@@ -0,0 +1,105 @@
+/* global chai */
+
+var React = require('react');
+var TestUtils = require('react/lib/ReactTestUtils');
+var _ = require('lodash');
+var expect = chai.expect;
+
+var Daily = require('../../../../app/components/chart/daily');
+
+require('tideline/css/tideline.less');
+require('../../../../app/core/less/fonts.less');
+require('../../../../app/style.less');
+window.localStorage.debug = true;
+
+describe('Daily', function () {
+  describe('render', function() {
```

Comment:
Don't worry about it at the moment - I started playing around with it locally and have non-pushed work in flight right now that I may get a chance to get back to later today. I realize now the issue is actually not with the tideline interface but interestingly maybe with PhantomJS's abilities to simulate DOM elements? Although it also seems that if we can figure out how to mock the inner `DailyChart` React component, then we'll also be able to get around this...